web/src/pages/CreateRoom/CreateRoom.tsx

```diff
@@ -0,0 +1,73 @@
+import { Component, createSignal, Show } from "solid-js";
+import Button from "@/components/Button";
+
+import styles from "./CreateRoom.module.scss";
+import useCreateRoom from "./hooks/useCreateRoom";
+
+const defaultNumberOfOptions = 5;
+
+const CreateRoom: Component = () => {
+	const { fields, updateField, submit } = useCreateRoom();
+
+	const handleSubmit = (event: Event): void => {
+		event.preventDefault();
+		submit();
```

Comment:
Should we go ahead and redirect to the room after it's created?

Reply:
- [ ] redirect to room after creation